assets/react/admin-dashboard/segments/filter.js

```diff
@@ -43,6 +43,16 @@ document.addEventListener('DOMContentLoaded', function () {
 			{ once: true },
 		);
 	}
+	const filterPaymentStatus = document.getElementById('tutor-backend-filter-payment-status');
+	if (filterPaymentStatus) {
+		filterPaymentStatus.addEventListener(
+			'change',
+			(e) => {
+				window.location = urlPrams('payment-status', e.target.value);
+			},
+			{ once: true },
+		);
+	}
```

Comment:
You can remove the if condition by using optional chaining.
```suggestion
		filterPaymentStatus?.addEventListener(
			'change',
			(e) => {
				window.location = urlPrams('payment-status', e.target.value);
			},
			{ once: true },
		);
```

BTW, why does this event is attached for once?